index.html

```diff
@@ -5,12 +5,85 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Bangers|Caveat|Courgette|Lobster&display=swap" rel="stylesheet">
     <title>Responsive Cake webpage</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
-    <!-- Add your markup here -->
+ <main>
+   <header>
+    <div class="Logo">
+        <img class="logoup" src="Yusuf Logo.png" alt="logo"/>
+        <p>best home made cake delivered at your door</p>
```

Comment:
As this forms part of your header, would this make more sense to be a `<h1>` (or a variation of this) element? Alternatively if this text is meant to always be with your img element, consider instead using `<figure>` and `<figcaption>`